tests/test_shape.py

```diff
@@ -0,0 +1,100 @@
+"""Test for the check_shape function."""
+
+import numpy as np
+import pytest
+from pytest_cases import parametrize_with_cases, parametrize, fixture
+from mrinufft import get_operator
+from case_trajectories import CasesTrajectories
+
+from mrinufft.operators.base import FourierOperatorBase
+
+from helpers import (
+    wrong_image_from_op,
+    wrong_kspace_from_op,
+    kspace_from_op,
+    image_from_op,
+    to_interface,
+    from_interface,
+    param_array_interface,
+)
+
+
+@fixture(scope="module")
+@parametrize(
+    "backend",
+    [
+        "torchkbnufft-gpu",
```

Comment:
Why only torchkbnufft-gpu? Please add others also if it was by mistake?

Reply:
I just put one at first and forgot to add others ones :)